python/README.md

```diff
@@ -1,21 +1,13 @@
 # truelayer-signing
 
-## Description
-
 Python package to produce & verify TrueLayer API requests signatures.
 
-## How to build
-
-Python: `poetry install`
-
-## Contributing
-Enable `black` and `flake-8` pre-commit hooks: `pre-commit install`
-
-## How to test
-- Type checking: `poetry run mypy ./truelayer_signing`
-
-- Test suite: `poetry run pytest -v tests/`
+## Install
+```
+pip install truelayer-signing
```

Comment:
I see that you've removed instructions for installing using `poetry`. Given that you still have `poetry.lock` and `pyproject.toml`, do you still expect people to install using `poetry`?

Reply:
Installing using poetry is not required as the lib is published and is accessible using pip https://pypi.org/project/truelayer-signing/

Reply:
In that case, should the poetry files be removed? Or will you keep them

Reply:
the poetry files are for the developers -- it's a replacement for the `setup.py` file.